fecgov/openFEC
# Remove deprecated endpoints

Delete the deprecated endpoints from the swagger documentation and their references from python files.

- Resolves #4024

Removed deprecated endpoints code and its references from following files:
1. webservices/args.py
2. webservices/rest.py

Deleted resource and pytest files:
1. tests/test_aggregates_by_committee_id.py
2. webservices/resources/aggregates_by_committeeID.py

## How to test the changes locally

- checkout branch  feature/4024-delete-deprecated-endpoints
- run pytest
- start server ./manage.py runserver
- open the swagger documentation in a browser (http://127.0.0.1:5000/)
- api endpoints under receipts, disbursements and independent expenditure as shown in the screen shots below are deprecated. They should not show in the swagger documentation on your local anymore. 

- Receipts section:
<img width="1111" alt="Screen Shot 2019-12-26 at 9 31 34 PM" src="https://user-images.githubusercontent.com/11650355/71497637-218dd400-2827-11ea-9137-4d2a2dad9d79.png">

- Disbursement section:
<img width="1080" alt="Screen Shot 2019-12-26 at 9 32 44 PM" src="https://user-images.githubusercontent.com/11650355/71497677-46824700-2827-11ea-8ef2-316290c201e6.png">

- Independent Expenditure section:
<img width="1206" alt="Screen Shot 2019-12-26 at 9 33 23 PM" src="https://user-images.githubusercontent.com/11650355/71497706-6285e880-2827-11ea-9812-edb37a69cf9b.png">

## Impacted areas of the application
List general components of the application that this PR will affect:

API endpoints under these 3 sections:
-  disbursements 
- receipts 
- independent expenditure
